seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSource.java

```diff
@@ -217,6 +225,11 @@ private PartitionParameter initPartitionParameterAndExtendSql(Connection connect
             }
             PartitionParameter partitionParameter =
                     initPartitionParameter(partitionColumn, connection);
+            if (partitionParameter == null) {
+                LOG.info(
+                        "The partition_column parameter is configured, but the result set is empty");
+                return null;
```

Comment:
is there has some more better way solve the problem, because return null is ugly

Reply:
sure, let me change this code to use optional to return